SU2_CFD/src/output/COutput.cpp

```diff
@@ -383,18 +368,11 @@ void COutput::LoadData(CGeometry *geometry, CConfig *config, CSolver** solver_co
 
 }
 
-void COutput::WriteToFile(CConfig *config, CGeometry *geometry, OUTPUT_TYPE format, string fileName){
+void COutput::WriteToFile(CConfig *config, CGeometry *geometry, OUTPUT_TYPE format, string fileName) {
 
   /*--- File writer that will later be used to write the file to disk. Created below in the "switch" ---*/
   CFileWriter *fileWriter = nullptr;
 
-  /*--- Set current time iter even if history file is not written ---*/
-  curTimeIter = config->GetTimeIter();
```

Comment:
I believe this was a fix for something, double check with git-blame please.

Reply:
yes, fix_UnstNames, I'll revert it.